### DISCUSSION TITLE
Is it possible to import any UE4-created material/asset

### DISCUSSION THREAD

**Discussion by schnuede** [carla-simulator/carla#7696] [General]
Hi,

i'm a newbie to CARLA and RTFM got me only so far.

Is it possible to import any (!) material or asset (like custom street signs with custom shaders) for simulation into Carla, or do I need to abide by the manipulation options presented in the wiki?

I am thinking about including retroreflective properties into some of the street signs according to national standards.

Thank you so much!

**Comment by ADASDevelopment** [comment]
You can transfer very easily in RoadRunner.
